### ISSUE TITLE
Update validate-bundle to include report filepath flag

### ISSUE THREAD

**Issue by riverma** [NASA-PDS/validate#194]
**Is your feature request related to a problem? Please describe.**
Our validation pipeline for InSight currently has a preset configuration file that hard-codes all validation parameters we want set against any given InSight bundle. We're using the `validate` tool currently, not the `validate-bundle` and we'd like to use the parallelization support present in the new `v1.21.0` `validate-bundle` script but can't currently pass our configuration file to the `validate-bundle` script.

**Describe the solution you'd like**
Add ability to pass a configuration file to the `validate-bundle` script similar to how a configuration file is supported with the underlying `validate` script

**Describe alternatives you've considered**
Pass all parameters on the command-line to `validate-bundle`. This doesn't seem to work, as passing the parameters at the command line yields a usage statement back from `validate-bundle`:

```
$ validate-bundle -R pds4.bundle --spot-check-data 500 -t /path/to/my/bundle/bundle.xml

usage: validate-bundle -t </path/to/bundle.xml> [-n <num-groups>]

  Script to run Validate Tool on a bundle parallelizing across 
  multiple cores. This script will split up the PDS4 products in 
  the bundle into multiple groups, and run in parallel across 
  multiple cores. By default,the software uses half the number of 
  available cores on the machine, or the -n flag can be used to 
  specify a custom number of cores.

  Dependencies:
      * GNU Parallels is installed (See https://www.gnu.org/software/parallel/)
      * Validate Tool is in your PATH
        % export PATH=${PATH}:/path/to/validate-x.x.x/bin

  Arguments: 

  -t, --target-bundle  </path/to/bundle.xml>  Indicate the bundle to be validated 
                                              by specifying the bundle.xml product.

  -n, --num-groups     <num-groups>           Optional argument to specify the number  
                                              of groups and cores to use for the validation.
                                              Default: half the number of cores on the machine
```

**Additional context**
Given that the above command-line equivalent did not work, there may be an additional bug here regarding passing other `validate` flags to `validate-bundle`

**Comment by jordanpadams** [comment]
@riverma what does your configuration file look like and how critical is it that it support a config file vs CLI options?

**Comment by jordanpadams** [comment]
@riverma additionally, the intent of this script is not to support all capabilities of validate, but rather to support an operational validation of an archive. spot checking content validation is really more for testing purposes than an operational "run for the record" to validate that the archive is valid.

I would recommend running this as `validate-bundle -t <path>`

**Comment by riverma** [comment]
> @riverma what does your configuration file look like and how critical is it that it support a config file vs CLI options?

Hi @jordanpadams - it's critical only if we can't leverage a command-line equivalent of the below configuration file:

```
validate.target = {{ data_delivery_folder }}
validate.report = {{ validate_tool_run_report_path }}
validate.rule = pds4.bundle
validate.spotCheckData = 500
```

Per your second comment, noted on the `validate.spotCheckData` field - thanks for mentioning that. I haven't tried sending in the `validate.report` or `validate.rule` flags via `validate-bundle` CLI, but we'd at a minimum need those two to be passable in the CLI version.

**Comment by jordanpadams** [comment]
@riverma 

```
validate.target = {{ data_delivery_folder }} >>> use the -t flag
validate.report = {{ validate_tool_run_report_path }} >>> see comments below
validate.rule = pds4.bundle >>> this is done by default
validate.spotCheckData = 500 >>> wontfix
```

Per the validation report, we can provide this, as a note, this software runs validate N number of times simultaneously and the output report takes all the reports from the N validate runs and consolidates them.
